skietbaan_fe/src/components/NavbarMenu.js

```diff
@@ -30,11 +43,11 @@ class NavbarMenu extends Component {
 
 	isHome() {
 		if (window.location.pathname.endsWith("/home")) {
-			return <img src={require('../components/navbar-icons/leaderboard-red.png')}
+			return <img src={leaderboardRed}
```

Comment:
Inconsistent indentations in file `skietbaan_fe/src/components/Login.js` you used single indentation here it is not I would suggest to stick to single indentation

Reply:
made it consisitent